### PR TITLE
DEVX-2519: Add link to ccloud-observability from other places

### PR DESCRIPTION
**Pull request by awalther28**
### Description 

https://confluentinc.atlassian.net/browse/DEVX-2519

_What behavior does this PR change, and why?_

Pages altered: docs for [ccloud-stack ](http://staging-docs-independent.confluent.io.s3-website-us-west-2.amazonaws.com/docs-platform/PR/484/6.1.0/tutorials/examples/ccloud/docs/ccloud-stack.html#additional-resources) and [beginner-cli](http://staging-docs-independent.confluent.io.s3-website-us-west-2.amazonaws.com/docs-platform/PR/484/6.1.0/tutorials/examples/ccloud/docs/beginner-cloud.html#additional-resources)

Added: see also tip for ccloud-observability

Docs staging: http://staging-docs-independent.confluent.io.s3-website-us-west-2.amazonaws.com/docs-platform/PR/484/6.1.0/overview.html

### Author Validation

_Describe the validation already done, or needs to be done, by the PR submitter._

- [x] Documentation

### Reviewer Tasks

_Describe the tasks/validation that the PR submitter is requesting to be done by the reviewer._
I validated the docs build, locally and with a staging doc. Reviewing the text changes/staging docs is all that's necessary.

- [x] Documentation

### CODE REVIEW

File: ccloud-observability/docs/ccloud-observability-tip.rst [hunk 1]
```diff
@@ -0,0 +1,2 @@
+
+.. seealso:: For an example that showcases client and Confluent Cloud metrics to monitor with various scenarios, see the :ref:`Observability for Apache Kafka® Clients to Confluent Cloud demo <ccloud-observability-index>`.
```
Comment (ybyzek) [inline ccloud-observability/docs/ccloud-observability-tip.rst]: @awalther28 feel free to wordsmith further.

```suggestion
.. seealso:: For an example that showcases how to monitor |ak-tm| client application and |ccloud| metrics, and steps through various failure scenarios to see how they are reflected in the provided metrics, see the :ref:`Observability for Apache Kafka® Clients to Confluent Cloud demo <ccloud-observability-index>`.
```

File: ccloud/docs/beginner-cloud.rst [hunk 2]
```diff
@@ -841,3 +841,6 @@ Additional Resources
    optimizing your |ak| client applications when using |ccloud|.
 
 -  See other :ref:`ccloud-demos-overview`.
+
+-  For an example that showcases client and Confluent Cloud metrics to monitor with various scenarios, see the :ref:`Observability for Apache Kafka® Clients to Confluent Cloud demo <ccloud-observability-index>`.
```
Comment (ybyzek) [inline ccloud/docs/beginner-cloud.rst]: See text above

**Review by ybyzek — APPROVED** [review]
LGTM (assuming all builds well)

**Comment by awalther28** [comment]
@ybyzek just verified that the docs build locally so will merge.

**Pull request merged**